app/controllers/bgeigie_imports_controller.rb

```diff
@@ -39,6 +39,12 @@ def fixdrive
     @bgeigie_import.fixdrive!
     redirect_to @bgeigie_import
   end
+  
+  def process_button
+    @bgeigie_import = scope.find(params[:id])
+    @bgeigie_import.process_button!
```

Comment:
Have you tried just calling `.process` here? Seems like the `.process_button!` method you've added is exactly the same as `.process`